### PR TITLE
Update rapid templates in docs

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

Reported on slack. These are outdated and not public to be updated if required. Added them to `pl_examples`.

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7

### CODE REVIEW

File: docs/source/starter/introduction.rst [hunk 1]
```diff
@@ -615,7 +615,7 @@ Before installing anything, use the following templates to try it out live:
         .. raw:: html
 
             <div style='width:150px;height:auto'>
-                <a href="https://colab.research.google.com/drive/1rHBxrtopwtF8iLpmC_e7yl3TeDGrseJL?usp=sharing>">
+                <a href="https://colab.research.google.com/github/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/basic_examples/rapid_templates/Rapid_prototyping_general_1.ipynb>">
```
Comment (kaushikb11) [inline docs/source/starter/introduction.rst:618]: Why not add it to lightning tutorials?

Comment (rohitgr7) [reply docs/source/starter/introduction.rst:618]: I thought it doesn't have anything much to provide, just some random dataset with simple code, so tutorials might not be worth it.

Comment (Borda) [reply docs/source/starter/introduction.rst:618]: btw, why is there the `_1` in the name?

Comment (rohitgr7) [reply docs/source/starter/introduction.rst:618]: Don't know. I'll update it with something better.

**Review by akihironitta — COMMENTED** [review]
Thanks for updating these notebooks! I'm in favour of adding this to lightning-tutorials though since notebooks can be quickly outdated as we don't have testing here. cc @Borda

**Comment by rohitgr7** [comment]
> Thanks for updating these notebooks! I'm in favour of adding this to lightning-tutorials though since notebooks can be quickly outdated as we don't have testing here. cc @Borda

we don't test other notebooks as well here. if we add them to Lightning tutorials, won't they come up inside the tutorials section in the docs and last time I checked we need to add them as scripts and notebooks are created on the fly.

**Review by akihironitta — APPROVED** [review]

**Comment by akihironitta** [comment]
@rohitgr7 @carmocca Just a small suggestion: Would you agree that we should cherry-pick this PR in `1.6.x`? Otherwise, the following docs won't have these corrected notebooks if I understand it correctly:
- https://pytorch-lightning.readthedocs.io/en/stable/
- https://pytorch-lightning.readthedocs.io/en/1.6.4/ (in the future)

**Comment by carmocca** [comment]
Fine with me

**Review by krshrimali — APPROVED** [review]
Just a small question, otherwise looks good to me! Thanks!

File: docs/source/starter/introduction.rst [hunk 2]
```diff
@@ -615,7 +615,7 @@ Before installing anything, use the following templates to try it out live:
         .. raw:: html
 
             <div style='width:150px;height:auto'>
-                <a href="https://colab.research.google.com/drive/1rHBxrtopwtF8iLpmC_e7yl3TeDGrseJL?usp=sharing>">
+                <a href="https://colab.research.google.com/github/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/basic_examples/rapid_templates/Rapid_prototyping_auto.ipynb>">
```
[the remark this answers is not in the document]
Comment (rohitgr7) [reply docs/source/starter/introduction.rst:618]: yes

**Review by Borda — COMMENTED** [review]
Why are we adding back heavy notebooks to this repo? :confused:

**Comment by rohitgr7** [comment]
> Why are we adding back heavy notebooks to this repo? 😕

heavy notebook? It's not that big.. notebooks generally show huge LOC. We need to keep it somewhere so that it can be easily managed/updated.

**Comment by Borda** [comment]
> heavy notebook? It's very minimal... 

that is usually how it starts :D

> notebooks generally show huge LOC. We need to keep it somewhere so that it can be easily managed/updated.

ok, so let's remove the inline image and use link to online one :)

File: pl_examples/basic_examples/rapid_templates/Rapid_prototyping_auto.ipynb [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
...lmsfsJ+QfOTJMh/BZk9zb+FqSPCNRb4NuXD3ZKk4nXZss3b5VIfjr/+BYFj2fI7bsX8PmGHclLYmNc5UROSPt6d/g8TdadDT+aEyGfw+4/fvEKwjjhhBPcZ7ajexXp8EGgaSR4HnhF5CbIUeEZgAO5BeQPfMi3QBSJLO0nhube8FN9lbsbJW1dlXwweaeM7pkpEITzDNpJEOQwCFXGXJQpH0/dKaE/qtz/kvDumkwkKNHqwtVnSy0JMSaH7yVYSTwChAysDBBDk5RFKUi64SUQwrFio3mAxLd+8NQI6RFCkHdglUYTyOCAt4AXQb4CvBJZEkYM2qmGOpGSgjr5c+keeXXYdhlybpNSx/3LTHgNzXkEfjT29VH58ufSctldyA+YaGvJfyW2YymMRBmCS1xq5cAgQAKOOJ1kI8lKkqZ8x0PX8o8UTw1ywHNgFYnkMDgQYvCdF3IWBwKHhBODikhdjUhBVq38/MlumdY/VwZ1SnO5gVjDi8HnprtfjJ5+e46sWFgqu/JqpY78YhLDBh2L/xWB1cN/zT4nFgGUgpwDFhFS5uDz4Rw+REKQcZN4VhyQwQOFwwEjBje4BpGqikbJ3bJX/vdKkVuJILzAA4hEEC5s6JQm4y/Pcv/lyuURKpKbR4g0MXbeEDhSEDiwxNCMIq5/eUm9/LuuSuY9uUNG9AjJA532X94cTh6hU5qMPj/D/U/K0PrmPMJB4CEcKcJg4zQEFIGkEIM2Rv6heHudrF+yR14elC+DO6e7JU7+yQz/T2L24G2yflm5lHZwHkH7a6+GwJGKQFKJQUGu3SuyI6dWfpq/W6bcnC1Tb8lxPxK7a3ut8EMtB0MeQftqr4bAkYhAhxCDA7pBpLqyUXZk18r2jBrZW2GEcCQKoI354ESg44ihGQ82JyVzg9LBOQ3WK0Pg4EKgw4nh4ILDemMIGAIgYMRgcmAIGAIBBIwYApDYCUPAEDBiMBkwBAyBAAJGDAFI7IQhYAgYMZgMGAKGQAABI4YAJHbCEDAEjBhMBgwBQyCAgBFDABI7YQgYAkYMJgOGgCEQQMCIIQCJnTAEDAEjBpMBQ8AQCCBgxBCAxE4YAoaAEYPJgCFgCAQQMGIIQGInDAFDwIjBZMAQMAQCCBgxBCCxE4aAIWDEYDJgCBgCAQSMGAKQ2AlDwBD4P9CuROTFaWXrAAAAAElFTkSuQmCC)"
```
Comment (rohitgr7) [inline pl_examples/basic_examples/rapid_templates/Rapid_prototyping_auto.ipynb:23]:
```suggestion
        "<img src=\"https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master/docs/source/_static/images/logo.png\" alt=\"PyTorch Lightning\" width=\"500\">"
```

File: pl_examples/basic_examples/rapid_templates/Rapid_prototyping_manual.ipynb [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
...lmsfsJ+QfOTJMh/BZk9zb+FqSPCNRb4NuXD3ZKk4nXZss3b5VIfjr/+BYFj2fI7bsX8PmGHclLYmNc5UROSPt6d/g8TdadDT+aEyGfw+4/fvEKwjjhhBPcZ7ajexXp8EGgaSR4HnhF5CbIUeEZgAO5BeQPfMi3QBSJLO0nhube8FN9lbsbJW1dlXwweaeM7pkpEITzDNpJEOQwCFXGXJQpH0/dKaE/qtz/kvDumkwkKNHqwtVnSy0JMSaH7yVYSTwChAysDBBDk5RFKUi64SUQwrFio3mAxLd+8NQI6RFCkHdglUYTyOCAt4AXQb4CvBJZEkYM2qmGOpGSgjr5c+keeXXYdhlybpNSx/3LTHgNzXkEfjT29VH58ufSctldyA+YaGvJfyW2YymMRBmCS1xq5cAgQAKOOJ1kI8lKkqZ8x0PX8o8UTw1ywHNgFYnkMDgQYvCdF3IWBwKHhBODikhdjUhBVq38/MlumdY/VwZ1SnO5gVjDi8HnprtfjJ5+e46sWFgqu/JqpY78YhLDBh2L/xWB1cN/zT4nFgGUgpwDFhFS5uDz4Rw+REKQcZN4VhyQwQOFwwEjBje4BpGqikbJ3bJX/vdKkVuJILzAA4hEEC5s6JQm4y/Pcv/lyuURKpKbR4g0MXbeEDhSEDiwxNCMIq5/eUm9/LuuSuY9uUNG9AjJA532X94cTh6hU5qMPj/D/U/K0PrmPMJB4CEcKcJg4zQEFIGkEIM2Rv6heHudrF+yR14elC+DO6e7JU7+yQz/T2L24G2yflm5lHZwHkH7a6+GwJGKQFKJQUGu3SuyI6dWfpq/W6bcnC1Tb8lxPxK7a3ut8EMtB0MeQftqr4bAkYhAhxCDA7pBpLqyUXZk18r2jBrZW2GEcCQKoI354ESg44ihGQ82JyVzg9LBOQ3WK0Pg4EKgw4nh4ILDemMIGAIgYMRgcmAIGAIBBIwYApDYCUPAEDBiMBkwBAyBAAJGDAFI7IQhYAgYMZgMGAKGQAABI4YAJHbCEDAEjBhMBgwBQyCAgBFDABI7YQgYAkYMJgOGgCEQQMCIIQCJnTAEDAEjBpMBQ8AQCCBgxBCAxE4YAoaAEYPJgCFgCAQQMGIIQGInDAFDwIjBZMAQMAQCCBgxBCCxE4aAIWDEYDJgCBgCAQSMGAKQ2AlDwBD4P9CuROTFaWXrAAAAAElFTkSuQmCC)"
```
Comment (rohitgr7) [inline pl_examples/basic_examples/rapid_templates/Rapid_prototyping_manual.ipynb:23]:
```suggestion
        "<img src=\"https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master/docs/source/_static/images/logo.png\" alt=\"PyTorch Lightning\" width=\"500\">"
```

**Comment by rohitgr7** [comment]
closing this. Looks like rapid templates are removed already in recent doc update.

**Pull request closed without merging**